static/webpack-config.js

```diff
@@ -127,11 +127,42 @@ module.exports = function () {
             }
           ],
         },
+        {
+          test: /\.html$/i,
+          use: [
+            {
+              loader: 'html-loader',
+              options: {
+                sources: {
+                  list: [
+                    {
+                      tag: 'link',
+                      attribute: 'href',
+                      type: 'src',
+                      filter: (tag, attribute, attributes) => {
+                        const isPreload = attributes.find(attr => {
+                          return attr.name === 'rel' && attr.value === 'preload';
+                        });
+                        const isFont = attributes.find(attr => {
+                          return attr.name === 'as' && attr.value === 'font';
+                        });
+                        return isPreload && isFont;
+                      }
+                    }
+                  ],
+                  urlFilter: (attribute, value) => {
+                    return /^[./]*static\/assets\//.test(value);
```

Comment:
Do we need to escape that forward slash inside the brackets? Regex101 is giving me an error there

Reply:
really? it works fine for me on regex101, maybe you weren't setting it to javascript?

Reply:
oh, that was it